.travis.yml

```diff
@@ -11,3 +11,11 @@ install:
 
 script:
   - npm run lint
+
+deploy:
+  provider: npm
+  email: "ben@daostack.io"
```

Comment:
this will ony work for you :-)

Reply:
I added it as Travis requires credentials of some account with publishing permissions, so I added mine.